CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/attribute/compatibility/AttrListAttributeDescriptionV0.java

```diff
@@ -1,223 +0,0 @@
-/*
- * Copyright 2010-2019 Australian Signals Directorate
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package au.gov.asd.tac.constellation.graph.attribute.compatibility;
```

Comment:
@cygnus-x-1 is this removed as a cleanup activity? Unfortunately I need all of these deprecated classes to remain to open old graphs that have written data with these attribute descriptions.

An idea I had was perhaps we could move all of these deprecated classes into a "Attributes(Deprecated)" module so that everyone else does not have to worry about them?

When I get an update site running this "deprecated" module can be turned off by default too to avoid everyone having to deal with these legacy formats.

Reply:
Yes - this was removed as part of cleaning up and normalising attributes. The issue here is that this attribute type is no longer supported (and can be easily broken), but because it's still 'registered' as an attribute it is available for use in the Attribute Editor, Delimited File Importer, etc. Rather than update the deprecated class, I figured it was easier to just remove it; after all that is the point of deprecation. Perhaps it's time to update those old graphs?

Reply:
Can do (eventually). For now though can we keep them for a bit longer but start with removing the `ServiceProvider` annotation so they don't appear in the views? Doing this type of cleanup makes sense now that we are in v2 and I'm OK with that - I'll just need the classes around for a bit longer rather than having to troll through git history 😄.

Feel free to add a big comment saying which ones you think are broken too as that will help me.